ddl/ddl.go

```diff
@@ -550,10 +550,8 @@ func (dc *ddlCtx) removeReorgCtx(jobID int64) {
 func (dc *ddlCtx) notifyReorgWorkerJobStateChange(job *model.Job) {
 	rc := dc.getReorgCtx(job.ID)
 	if rc == nil {
-		logutil.BgLogger().Error("cannot find reorgCtx", zap.Int64("jobID", job.ID))
```

Comment:
Why remove these two logs?

Reply:
seams not necessary, I'd like to change it to be debug but not remove it.